third-culture-software/bhima
# Shipment container updates

Added weights to shipment containers.   Refactored the shipment document and added a new shipment manifest based or requests of likely near-term users.

Closes https://github.com/IMA-WorldHealth/bhima/issues/6754
Closes https://github.com/IMA-WorldHealth/bhima/issues/6739

**TESTING**
- Use bhima_test
- Go to: Shipments > New Shipment
- Create 3 shipments: 
   - one with no containers
   - one with 1 container (Use the [Containers] menu to set the weight of the container)
   - one with 2 or more containers (Use the [Containers] menu to set the weight of the containers)
- Go to the shipments registry (Shipments > Shipments Registry)
   - Using the action menu, show the "Document" and "Manifest" for each shipment
   - Verify quantities, weights, etc
 


-----
[View rendered docs/en/stock-management/movement.shipments.md](https://github.com/jmcameron/bhima/blob/shipment-container-updates/docs/en/stock-management/movement.shipments.md)
